### PR TITLE
Revert "[Windows] Fix default corner radius on Button"

### LINKED ISSUE #10644: Validate Windows buttons have round corners
### Description

Last I saw something broke the default style so it reverted back to win 10 themes.

### Steps to Reproduce

Run windows

### Link to public reproduction project repository

maui

### Version with bug

Unknown/Other (please specify)

### Last version that worked well

Unknown/Other

### Affected platforms

Windows

### Affected platform versions

all

### Did you find any workaround?

_No response_

### Relevant log output

_No response_

### PR DESCRIPTION
**Pull request by tj-devel709**
Reverts dotnet/maui#9515

dotnet/maui#9515 Allows all the platforms to have the same default button border radius, but we want to keep the Windows default rounded corner style on buttons. Thanks @mattleibow and @PureWeen for spotting this!
Also peer programmed with @rachelkang!

Before the revert (no rounded corners):
Screen Shot 2022-10-12 at 3 34 27 PM

After the revert (rounded corners):
Screen Shot 2022-10-13 at 10 43 38 AM

Fixes #10644

### CODE REVIEW

**Review by jsuarezruiz — APPROVED** [review]

File: src/Core/src/Platform/Windows/ButtonExtensions.cs [hunk 1]
```diff
@@ -50,10 +50,10 @@ public static void UpdateStrokeThickness(this Button platformButton, IButtonStro
 
 		public static void UpdateCornerRadius(this Button platformButton, IButtonStroke buttonStroke)
 		{
-			var radius = buttonStroke.CornerRadius > -1 ? buttonStroke.CornerRadius : 0;
+			var radius = buttonStroke.CornerRadius;
 
 			if (radius >= 0)
-				platformButton.Resources.SetValueForAllKey(CornerRadiusResourceKeys, WinUIHelpers.CreateCornerRadius(radius));
+				platformButton.Resources.SetValueForAllKey(CornerRadiusResourceKeys, WinUIHelpers.CreateCornerRadius(buttonStroke.CornerRadius));
```
Comment (mattleibow) [inline src/Core/src/Platform/Windows/ButtonExtensions.cs:56]: Should we use the radius field instead of getting the value again?

**Comment by rachelkang** [comment]
Fixes #10644

File: src/Core/src/Platform/Windows/ButtonExtensions.cs (hunk 1 above)
Comment (tj-devel709) [reply src/Core/src/Platform/Windows/ButtonExtensions.cs]: Addressed!

**Comment by tj-devel709** [comment]
I added a small test to make sure the radius of the WinUI does not get set back to zero. 
Small photo added since I didn't realize the default button corner radius set is -1 which is rounded as well!
Screen Shot 2022-10-13 at 9 34 06 PM

File: src/Core/tests/DeviceTests/Handlers/Window/WindowHandlerTests.Windows.cs [hunk 2]
```diff
@@ -161,6 +161,13 @@ public async Task WindowSupportsEmptyPage_Platform()
 			});
 		}
 
+		[Fact(DisplayName = "Rounded Maui Buttons Default Style")]
+		public void RoundedCornersMauiButtons()
```
Comment (mattleibow) [inline src/Core/tests/DeviceTests/Handlers/Window/WindowHandlerTests.Windows.cs:165]: Could this be moved to the button handler tests?

**Comment by mattleibow** [comment]
The value of -1 means default rounding. So whatever the OS decides.

**Review by mattleibow — APPROVED** [review]

**Pull request merged**